tensorboard/components/tensor_widget/tensor-widget.html

```diff
@@ -0,0 +1,18 @@
+<!--
+@license
+Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<script src="tensor-widget.js"></script>
```

Comment:
FYI: it all depends on the build system but if you are going to use Vulcanization (as part of TensorBoard build), you will need another line for the dependencies of the tensor-widget.js, types.js. This is currently working okay only because TSC generates no JS file since it doesn't need one (won't be true the moment you have a [abstract] class).

Reply:
Yeah, I'm targeting Vulcanization right now. I added the files according to your suggestion.Thanks.